src/command/Parser.java

```diff
@@ -2,6 +2,7 @@
 
 import common.DukeException;
 import common.TaskList;
+import task.*;
```

Comment:
Maybe for better style only importing the necessary packages into a class would be better.